image/histogram_equalisation/testdriver.py

```diff
@@ -0,0 +1,56 @@
+import matplotlib
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.